README.md

```diff
@@ -123,7 +123,7 @@ else
 }
 ```
 
-Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a RememberMeID.
+Where `yourUserSearchFunction` is a piece of logic in your app that is supposed to find a user, given a RememberMeID or ParentRememberMeID.
```

Comment:
I think this should be `ParentRememberMeId` for consistency.

Reply:
To be honest, I prefer using caps when we're using a shortened name (Go's rubbing off on me - this is something they enforce!). But since this is implemented as `Id` in other SDKs, I can change to that

Reply:
I will change `ReceiptID` too for consistency, as that is currently upper case